libs/sdk-ui-filters/src/DateFilter/DateFilter.tsx

```diff
@@ -123,10 +124,28 @@ export class DateFilter extends React.PureComponent<IDateFilterProps, IDateFilte
         };
     };
 
+    private static checkInitialFilterOption = (filterOption: DateFilterOption) => {
+        if (isAbsoluteDateFilterForm(filterOption) && !(filterOption.from && filterOption.to)) {
+            // eslint-disable-next-line no-console
+            console.warn(
+                "The default filter option is not valid. Values 'from' and 'to' from absoluteForm filter option must be specified.",
+            );
+        }
+
+        if (isRelativeDateFilterForm(filterOption) && !(filterOption.from && filterOption.to)) {
```

Comment:
This is not correct, `from: 0` and `to: 0` are valid values and they should not trigger the warning. You should probably use `isNil` from lodash here.

Reply:
isNil reviewed and used to allow '0' as a valid default filter value